CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### changed
+
+- Support for `starlette>=0.13`, Using `on_startup` Router, and, Router's lifespan.
```

Comment:
Good idea to add a changelog entry in this PR, just a suggestion on the formatting.

```suggestion
### Changed

- Upgrade Starlette to `starlette==0.13.*`. (Pull #106)
```

Note: I somewhat agree with the "Changed" denomination here, but to be clear we'll probably release this as 0.8.0.